package.json

```diff
@@ -72,6 +72,7 @@
 		"raw-body": "^2.5.2",
 		"tsx": "^4.7.0",
 		"typescript": "^5.2.2",
+		"undici-types": "^6.6.2",
```

Comment:
Where is this used?

Reply:
It is used here: https://github.com/sindresorhus/ky/blob/cb89702ca488717a8b2021522f5ffe05a3c01a99/source/types/request.ts#L1.

I got a warning about the library not being found.

Reply:
I removed the dependency since it is no needed from [v1.2.2](https://github.com/sindresorhus/ky/compare/v1.2.1...v1.2.2).